__tests__/index.test.ts

```diff
@@ -4,8 +4,21 @@ import * as scope from '../src/scope';
 // @ts-ignore
 import { acquireLockSpy, releaseLockSpy } from 'browser-tabs-lock';
 
+jest.mock('browser-tabs-lock');
```

Comment:
```suggestion
```

Reply:
What does the hyphen prefix do here?

Reply:
Oh, I was looking at this diff on my phone, it wasn't obvious it was part of the diff markup 😄

Reply:
Have removed it manually.